lib/ansible/modules/cloud/amazon/elasticache_parameter_group.py

```diff
@@ -156,8 +156,10 @@ def make_current_modifiable_param_dict(module, conn, name):
     modifiable_params = {}
 
     for param in parameters:
-        if param["IsModifiable"] and ("AllowedValues" and "ParameterValue") in param:
-            modifiable_params[param["ParameterName"]] = [param["AllowedValues"], param["DataType"], param["ParameterValue"]]
+        if param["IsModifiable"]:
+            modifiable_params[param["ParameterName"]] = [param["AllowedValues"]] if "AllowedValues" in param else [None]
```

Comment:
Instead of inline ifs, you can use `[param.get("AllowedValues")]` instead and that will give you a list containing `None` if there are no allowed values.